gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -0,0 +1,265 @@
+"""Portfolio View"""
+__docformat__ = "numpy"
+
+from datetime import datetime, timedelta
+from io import BytesIO
+import os
+
+import pandas as pd
+from tabulate import tabulate
+from reportlab.lib.pagesizes import letter
+from reportlab.pdfgen import canvas
+from reportlab.lib.utils import ImageReader
+from matplotlib import pyplot as plt
+import matplotlib.ticker as mtick
+
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.portfolio import yfinance_model
+from gamestonk_terminal.portfolio import reportlab_helpers
+
+
+def load_info():
+    """Prints instructions to load a CSV"""
+    text = """
+In order to load a CSV do the following:
+
+1. Add headers to the first row, below is data for each column:\n
+\t1. Identifier for the asset (such as a stock ticker)
+\t2. Type of asset (stock, bond, option, crypto)
+\t3. The volume of the asset transacted
+\t4. The buy date in yyyy/mm/dd
+\t5. The Price paid for the asset
+\t6. Any fees paid during the transaction
+\t7. A premium paid or received if this was an option
+\t8. Whether the asset was bought (covered) or sold (shorted)\n
+2. Place this file in gamestonk_terminal/portfolio/portfolios\n
+        """
+    print(text)
+
+
+def show_df(df: pd.DataFrame, show: bool) -> None:
+    """
+    Shows the given dataframe
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        The dataframe to be shown
+    show : bool
+        Whether to show the dataframe index
+    """
+
+    if gtff.USE_TABULATE_DF:
```

Comment:
I think you should do 2 things here:
- Drop empty columns since they don't add anything
- Replace "nan" by empty string: ""